karapace/serialization.py

```diff
@@ -225,6 +227,25 @@ async def close(self):
         await self.client.close()
 
 
+def get_subject_name(
```

Comment:
I see that you only moved here code around, not changing. However now looking at this, is it correct code at all? For me it looks it takes only namespace of record, but is missing record name itself? `naming_strategy` functions take `record_name` as argument, not `namespace`. This is also reflected in the test you added. Are those expected values correct in that test is relevant question in this context.

Reply:
> is it correct code at all? For me it looks it takes only namespace of record, but is missing record name itself? 
good point, I've just moved stuff around and created a test to keep the behaviour. We need to find a real case where this is done and compare it to our implementation, I'm looking for it now